appserver/web/web-glue/src/main/java/com/sun/enterprise/web/VirtualServer.java

```diff
@@ -1166,6 +1166,7 @@ void configureErrorPage() {
                 _logger.log(Level.WARNING, LogFacade.NULL_VIRTUAL_SERVER_PROPERTY, getID());
                 continue;
             }
+            propValue = propValue.replace("\"", "");
```

Comment:
I think the purpose was to remove the quotes from beginning and end, not all of them.
I would expect some test, if there is a quote char at the first and last place and then remote them.

Reply:
also agree with this comment will suggest to add some king of integration tests for this

Reply:
Added a test